apps/condo/domains/meter/components/MeterReportingPeriodForm.tsx

```diff
@@ -153,15 +160,18 @@ export const MeterReportingPeriodForm: React.FC<IMeterReportingPeriodForm> = ({
         error: periodsLoadingError,
     } = MeterReportingPeriod.useObjects({
         where: {
-            property_is_null: false,
             organization: { id: organizationId },
         },
     },
     {
         fetchPolicy: 'network-only',
     })
 
-    const search = useMemo(() => searchOrganizationPropertyWithoutPropertyHint(organizationId, reportingPeriods.map(period => period.property.id)),
+    const hasOrganizationPeriod = Boolean(reportingPeriods.find(period => period.property === null && period.organization !== null))
```

Comment:
in case if property === undefined, then condition return false, but actually it should be true.
please use utils for such conditions (isNil(property) or !isNil(property))